src/components/dashboard/main-content/course-enrollments/course-cards/CourseEnrollmentsContext.jsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
```

Comment:
[suggestion] Might make sense to move this file up a directory, such that it's a child of `course-enrollments` directory vs. `course-cards`, as this context doesn't really pertain to the course cards themselves, but rather wraps the course enrollments.